### DISCUSSION TITLE
Eth lost in the wormhole

### DISCUSSION THREAD

**Discussion by ilgnocco** [wormhole-foundation/wormhole#512] [Problems / bugs] [answered]
Hello, 

I tried transferring Eth from Metamask to Solana (Phantom). It was successful, but the coin does not appear in my Phantom wallet under the Wrapped Eather (Wormhole) section. On explorer.solana.com I can see the address, mint, owner, and token total, but I do not know how to access the tokens since they do not appear in my wallet. What can I do?

-Ilgnocco

**Comment by leoluk** [comment, marked as answer]
Please provide more details about your transfer (tx hashes, screenshots?)
